TutorLizard.BusinessLogic/Data/IStudentDataAccess.cs

```diff
@@ -1,5 +1,12 @@
-﻿namespace TutorLizard.BusinessLogic.Data;
+﻿using TutorLizard.BusinessLogic.Models;
+
+namespace TutorLizard.BusinessLogic.Data;
 
 public interface IStudentDataAccess
 {
+    AdRequest CreateAdRequest(int adId, object value, bool isAccepted, string message);
+    ScheduleItemRequest CreateScheduleItemRequest(int scheduleItemId, int value);
+    IEnumerable<Ad> GetAcceptedUserAds(object userId);
+    object GetAllAds();
```

Comment:
Change return type from object (you almost never want to use object) to what you actually want - `List<Ad>`.